tests/testthat/test_standarize_call_formals.R

```diff
@@ -64,3 +64,25 @@ test_that("When an invalid function passed (i.e., corrupt language object)", {
   testthat::expect_equal(
     call_standardise_formals(user), user)
 })
+
+
+test_that("Standarize call with include_defaults = FALSE", {
+  library(readr)
```

Comment:
Is there another example that can be tested? 

CRAN will require `readr` to be installed. I'd rather not take a dependency on it.

Reply:
because it's on test/ I think it will only be in Suggest. Not in the Import field of DESCRIPTION file.

I will try to find another example.